Content.Shared/Entry/EntryPoint.cs

```diff
@@ -48,6 +50,9 @@ public override void PostInit()
             // fake lag rand leads to messages arriving out of order. Sadly, networking is not robust enough, so for now
             // just leaving this disabled.
             // configMan.OverrideDefault(CVars.NetFakeLagRand, 0.01f);
+
+            if (IoCManager.Resolve<INetManager>().IsServer)
+                configMan.OverrideDefault(CCVars.GameMap, "Dev");
```

Comment:
This should go into the `development` config preset, not be code like this.

Reply:
ah right, I forgot those exist now